chapters/synchronous.tex

```diff
@@ -1188,25 +1203,24 @@ \subsection{Solver Methods}\label{solver-methods}
 If a tool does not support the integration method defined in a model, typically a warning message is printed and the method is changed to \lstinline!"External"!.
 \end{nonnormative}
 
-If the solver method is \lstinline!"External"!, then the partition associated with this method is integrated by the simulation environment for an interval of length of
+If the solver method is \lstinline!"External"!, then the sub-partition associated with this method is integrated by the simulation environment for an interval of length of
 \lstinline!interval()! using a solution method defined in the simulation environment.
 
 \begin{nonnormative}
-An example of such a solution method could be to have a table of the clocks that are associated with discretized continuous-time partitions and a method selection per clock.
+An example of such a solution method could be to have a table of the clocks that are associated with discretized partitions and a method selection per clock.
 In such a case, the solution method might be a variable step solver with step-size control that integrates between two clock ticks.
 The simulation environment might also combine all partitions associated with method \lstinline!"External"!, as well as all continuous-time partitions, and integrate them together with the solver selected by the simulation environment.
 \end{nonnormative}
 
-If the solver method is \emph{not} \lstinline!"External"!, then the partition is
-integrated using the given method with the step-size \lstinline!interval()!.
+If the solver method is \emph{not} \lstinline!"External"!, then the sub-partition is integrated using the given method with the step-size \lstinline!interval()!.
 
 \begin{nonnormative}
 For a periodic clock, the integration is thus performed with fixed step size.
 \end{nonnormative}
 
 The solvers are defined with respect to the underlying ordinary
 differential equation in state space form to which the continuous-time
-partition can be transformed, at least conceptually ($t$ is time,
+sub-partition can be transformed, at least conceptually ($t$ is time,
```

Comment:
Still non-sentence-based line breaks here (and GitHub won't let me suggest a fix).

Reply:
This was corrected in a slightly more verbose way.
Hope it is ok.

Reply:
Yes.  Resolving.